src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -1,5 +1,22 @@
 package checkers.inference;
 
+import checkers.inference.dataflow.InferenceAnalysis;
```

Comment:
Why change the original order of imports? It makes hard to see which imports are newly added, which are just reordered.